app/controllers/post_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostController < ApplicationController
+end
```

Comment:
- [ ] Kindly make sure the `posts_controller` has both `#index` and `#show` actions(It's okay for the actions to be empty for now) but make sure to add them for the project requirement.

- [ **OPTIONAL** ] **RECOMMENDED:** Also make sure you update the `app/controllers/post_controller.rb` file name to `app/controllers/posts_controller.rb`, the class name `PostController` to `PostsController`, and the `app/views/post/` directory to ``app/views/post/`` as they raise the below error when trying to test them:
![Screenshot from 2022-11-08 05-45-16](https://user-images.githubusercontent.com/83042742/200470414-c3387a32-5edf-48c6-87ff-79d18dfa493e.png)

Reply:
I added the index and show to the controllers, and also I added the s to posts.